src/main/drivers/max7456.c

```diff
@@ -312,8 +321,15 @@ void max7456_dma_irq_handler(dmaChannelDescriptor_t* descriptor)
     if (DMA_GET_FLAG_STATUS(descriptor, DMA_IT_TEIF)) {
         DMA_CLEAR_FLAG(descriptor, DMA_IT_TEIF);
     }
-}
 
+    if (DMA_GET_FLAG_STATUS(descriptor, DMA_IT_FEIF)) {
```

Comment:
Yeah, I wanted to flag errors for debugging. I might leave it there for a while; I'm thinking of creating an SPI DMA agent that takes care of devices that shares a single SPI bus. The agent may return some meaningful error codes.